doc/generated/functions.gen

```diff
@@ -4384,6 +4384,24 @@ the tool object, previously it did not return
 </para>
 </listitem>
   </varlistentry>
+  <varlistentry id="f-ValidateOptions">
+    <term><function>ValidateOptions</function>()</term>
+    <term><replaceable>env</replaceable>.<methodname>ValidateOptions</methodname>()</term>
+    <listitem><para>
+                Check that all the options provided by the user on the command line are either defined by SCons itself
```

Comment:
Suggest: drop "by the user".  In most other places I think the docs have tended to use "specified" instead of "provided", although I don't much care for the word in this context. Just sayin'.